partials/content-single.php

```diff
@@ -67,15 +67,15 @@
 	if ( $display_about_the_author && $authors ) {
 		// TODO: add better check to display About the Authors section if at least one author has more than just a name & profile pic?
 		?>
-		<div class="contributors">
+		<section class="contributors">
```

Comment:
Changed from div to section, since I felt like section had relevant semantic meaning here. Open to suggestions, however.